### DISCUSSION TITLE
Treshold for minimal motion

### DISCUSSION THREAD

**Discussion by R0yk3** [blakeblackshear/frigate#5663] [Ideas]
Hello, First of all i want to say GREAT JOB for this software.
I have set up 2 camera's and it was a bit of a fiddle but now they work.
They work so good that i dare to ask this question ;)

i have a camera in my house, which turns on when nobody is at home. This way we have some privacy and i can keep an eye on the dogs. the dog filter works nice.
Now i have 1 dog who likes to lay down in the middle of the room and sleep there. It is detetcted but Frigate keeps filming the dog while sleeping in the middle of the room. Is it possible to set a maximimum clip lenght, or when the object has minimal movement it stops filming. Like the example with the parked car, but i don't want to ' black out'  the middle of my room. 
I would like to see when he takes his spot, lies down and sleep +1 minute, and when he wakes takes another spot lies down there.. etc. 
Hope you understand what i mean.

**Comment by NickM-27** [comment]
This is what recording retention modes are forhttps://docs.frigate.video/configuration/record#what-do-the-different-retain-modes-mean
